cmd/bosun/expr/funcs.go

```diff
@@ -360,7 +360,9 @@ func Filter(e *State, T miniprofiler.Timer, series *Results, number *Results) (*
 	for _, sr := range series.Results {
 		for _, nr := range number.Results {
 			if sr.Group.Subset(nr.Group) || nr.Group.Subset(sr.Group) {
-				ns = append(ns, sr)
+				if nr.Value.Value().(Number) != 0 {
```

Comment:
@captncraig I'm aware but throughout the funcs.go we don't check them since the map[string]parse.Func{... ensures we won't get there.